docs/source/reference/auth.rst

```diff
@@ -16,11 +16,19 @@ To add a token to the execution of a wis2box process, use the following command:
 
 .. code-block:: bash
 
-    wis2box auth add-token --path processes/wis2box myexecutiontoken
+    wis2box auth add-token --path processes/wis2box
```

Comment:
Do we want to add that they can specify a token as well?

Reply:
Yes I can add this to the docs